src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -31,6 +31,9 @@ class NUnit extends ConventionTask {
     def test
     def testList
 
+    Map<String, Object> custom = [:]
+    Map<String, Object> env = [:]
```

Comment:
Can you please update the README.md sample, so everybody will know about these two properties?